File: TRNMNT.Core/Model/Participant/ParticipantDdlModel.cs
using System.Collections.Generic;
using TRNMNT.Core.Model.Category;
using TRNMNT.Core.Model.Team;
using TRNMNT.Core.Model.WeightDivision;

namespace TRNMNT.Core.Model.Participant
{
    public class ParticipantDdlModel
    {
        public IEnumerable<TeamModelBase> Teams { get; set; }
        public IEnumerable<CategoryModelBase> Categories { get; set; }
        public IEnumerable<WeightDivisionModel> WeightDivisions { get; set; }
    }
}

File: TRNMNT/ClientApp/app/core/model/participant-ddl.model.ts
import { TeamModelBase } from './team.model';
import { CategoryModelBase } from './category.models';
import { WeightDivisionModel } from './weight-division.models';

export class ParticipantDdlModel {
    categories: CategoryModelBase[];
    weightDivisions: WeightDivisionModel[];
    teams: TeamModelBase[];

    constructor() {
        this.categories = [];
        this.weightDivisions = [];
        this.teams = [];
    }
}